### PR TITLE
FBI Hate Crime publications 11-14

### PR DESCRIPTION
**Pull request by Abilityguy**
Artifacts and Scripts used for the FBI Hate Crime Tables 11-14 import.
This PR follows the same directory structure as #622

### CODE REVIEW

File: scripts/fbi/hate_crime/table14/preprocess.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    df['agency'] = df['agency'].str.strip()
+
+    # Keep only cities
+    df = df[df['agency type'] == 'Cities']
+
+    df['state'] = df['state'].str.title()
+    df['state'] = df['state'].str.strip()
+
+    df = df[df['agency'].notna()]
+    df['agency'] = df['agency'].str.replace(r'\n', ' ', regex=True)
```
Comment (ajaits) [inline scripts/fbi/hate_crime/table14/preprocess.py:311]: can be moved to line 303?

File: scripts/fbi/hate_crime/table14/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+    df['agency'] = df['agency'].str.replace(r'\s+', ' ', regex=True)
```
Comment (ajaits) [inline scripts/fbi/hate_crime/table14/preprocess.py:312]: already done in line 301?

**Review by ajaits — APPROVED** [review]
Minor comments, o/w looks good.

I did't realize tests can use gitLFS files.

File: scripts/fbi/hate_crime/table14/preprocess.py (hunk 2 above)
Comment (Abilityguy) [reply scripts/fbi/hate_crime/table14/preprocess.py]: Done.

File: scripts/fbi/hate_crime/table14/preprocess.py (hunk 1 above)
Comment (Abilityguy) [reply scripts/fbi/hate_crime/table14/preprocess.py]: Done.

**Comment by Abilityguy** [comment]
Have made review fixes. 
Additionally, testdata is changed from Git LFS files to static Git files.

**Pull request merged**